### PR TITLE
fix(apple): Clarify pre-warm start

### PR DESCRIPTION
**Pull request by philipphofmann**
Mention since when we stop reporting pre-warm starts and point
out the GH issue.

### CODE REVIEW

File: src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 1]
```diff
@@ -48,11 +48,20 @@ SentrySDK.start { options in
 This feature is available for iOS, tvOS, and Mac Catalyst.
 
 The App Start Instrumentation provides insight into how long your application takes to launch. It adds spans for different phases, from the application launch to the first auto-generated UI transaction.
-To enable this feature enable `AutoUIPerformanceTracking`. The SDK differentiates between a cold and a warm start, but doesn't track hot starts/resumes.
+To enable this feature enable `AutoUIPerformanceTracking`.
```
Comment (imatwawana) [inline src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx:51]:
```suggestion
To enable this feature, enable `AutoUIPerformanceTracking`.
```

File: src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 2]
```diff
@@ -48,11 +48,20 @@ SentrySDK.start { options in
...
+
+<Alert level="info" title="Important">
+
+Since iOS 15, the system might decide to [pre-warm](https://developer.apple.com/documentation/uikit/app_and_environment/responding_to_the_launch_of_your_app/about_the_app_launch_sequence#3894431) your app by already creating the process before the user opens it. In such cases, we can't reliably measure the app start and drop it since [sentry-cocoa 7.18.0](https://github.com/getsentry/sentry-cocoa/releases/tag/7.18.0). We are working on a fix for this. Follow the [GH issue](https://github.com/getsentry/sentry-cocoa/issues/1897) for more details.
```
Comment (imatwawana) [inline src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx:55]: Do we need to say "decide to"? Can we just leave that out?
Also, I made another edit around the language about us dropping the app start to try to make that clearer

```suggestion
Starting with iOS 15, the system might [pre-warm](https://developer.apple.com/documentation/uikit/app_and_environment/responding_to_the_launch_of_your_app/about_the_app_launch_sequence#3894431) your app by creating the process before the user opens it. In such cases, we can't reliably measure the app start, so we drop it as of [sentry-cocoa 7.18.0](https://github.com/getsentry/sentry-cocoa/releases/tag/7.18.0). We are working on a fix for this. Follow the [GitHub issue](https://github.com/getsentry/sentry-cocoa/issues/1897) for more details.
```

**Review by imatwawana — APPROVED** [review]
Made some small wording changes for clarity

**Comment by philipphofmann** [comment]
Thanks @imatwawana.

**Pull request merged**